modules/core/src/test/java/org/apache/ignite/internal/metric/SystemViewSelfTest.java

```diff
@@ -259,6 +263,114 @@ public void testCacheGroupsView() throws Exception {
+                .setGroupName("group2")
+                .setEagerTtl(false)
+                .setExpiryPolicyFactory(CreatedExpiryPolicy.factoryOf(new Duration(TimeUnit.MILLISECONDS, 500L)));
+
+            CacheConfiguration<Long, Long> withoutGrpCache = new CacheConfiguration<Long, Long>(withoutGrpCacheName)
+                .setExpiryPolicyFactory(CreatedExpiryPolicy.factoryOf(new Duration(TimeUnit.MILLISECONDS, 500L)));
+
+            CacheConfiguration<Long, Long> dfltCache = new CacheConfiguration<Long, Long>(dfltCacheName)
+                .setGroupName("group3");
+
+            g.createCache(eternalCache);
+            g.createCache(createdCache);
+            g.createCache(eagerTtlCache);
+            g.createCache(withoutGrpCache);
+            g.createCache(dfltCache);
+
+            SystemView<CacheView> caches = g.context().systemView().view(CACHES_VIEW);
+
+            for (CacheView row : caches) {
+                switch (row.cacheName()) {
+                    case "defaultCache":
+                    case "eternalCache":
+                        assertEquals("No", row.expiryPolicyEntry());
+
+                        g.cache(row.cacheName()).put(0, 0);
+
+                        assertEquals("No", row.expiryPolicyEntry());
+
+                        g.cache(row.cacheName())
+                            .withExpiryPolicy(new CreatedExpiryPolicy(new Duration(TimeUnit.MILLISECONDS, 200L)))
+                            .put(1, 1);
+
+                        assertEquals("Yes", row.expiryPolicyEntry());
+                        assertTrue(waitForCondition(() -> row.expiryPolicyEntry().equals("No"), getTestTimeout()));
+
+                        break;
+
+                    case "createdCache":
+                        assertEquals("No", row.expiryPolicyEntry());
+
+                        g.cache(createdCacheName).put(0, 0);
+
+                        assertEquals("Yes", row.expiryPolicyEntry());
+                        assertTrue(waitForCondition(() -> row.expiryPolicyEntry().equals("No"), getTestTimeout()));
+
+                        g.cache(createdCacheName)
+                            .withExpiryPolicy(new ModifiedExpiryPolicy(new Duration(TimeUnit.MILLISECONDS, 200L)))
+                            .put(1, 1);
+
+                        assertEquals("Yes", row.expiryPolicyEntry());
+                        assertTrue(waitForCondition(() -> row.expiryPolicyEntry().equals("No"), getTestTimeout()));
+
+                        g.cache(eagerTtlCacheName).put(2, 2);
+                        assertEquals("No", row.expiryPolicyEntry());
+
+                        break;
+
+                    case "eagerTtlCache":
+                        assertEquals("Unknown", row.expiryPolicyEntry());
+
+                        break;
+
+                    case "withoutGrpCache":
+                        assertEquals("There is no cache group related data", row.expiryPolicyEntry());
```

Comment:
It's not correct, here cache group is created with `sharedGroup == false` flag.

Reply:
Fixed it